_posts/2016-10-07-our-jekyll-workflow.md

```diff
@@ -0,0 +1,903 @@
+---
+layout: post
+title: Our Jekyll Workflow
+date: 2016-10-07
+author: Anne Tomasevich
+tags: performance jekyll javascript css sass front-end-dev
+summary: |
+  How we improved our company site's performance with a better Jekyll workflow
+featured_image: "/blog/test-tubes.jpg"
+featured_image_alt: "Test tubes with brightly color liquids inside"
+---
+
+It's hard to believe it's been over a year and a half since our site's
+[inaugural blog post](http://localhost:3000/2015/04/01/building-our-site.html)
+(written just two months after my career change into web development!) It's been
+great fun building our site and adding content thanks to the power and simplicity
+of [Jekyll](https://jekyllrb.com/). We recently deployed a series of changes to
+optimize our CSS, JS, and images and, in doing so, finally moved away from using
+built-in `jekyll` commands to build the site. Instead, we're using
+[gulp](http://gulpjs.com/), a task runner — or, as they put so nicely on their
+website, a "streaming build system". In this post I'll go over our motivations
+for this change, how we integrated gulp with Jekyll, and the awesome results!
+
+We based a lot of our gulp workflow on
+[this excellent post](https://robwise.github.io/blog/jekyll-and-gulp) by Rob Wise.
+Our workflow includes a few additional features like:
+
+- Separate gulp tasks for building the site locally,
+for testing purposes, and for production
+- Some updated gulp plugins
+- Inlining of critical CSS
+- Some further improvements to responsive images
+
+...but I'd definitely recommend his post and you'll see some of his code and
+architecture throughout this article.
+
+To see all of our current code, pop over to our
+[GitHub repo](https://github.com/savaslabs/savaslabs.github.io).
+
+## Motivation
+
+We've wanted to make some improvements to our site's performance for a while,
+but our ultimate motivator was our not-so-great Google PageSpeed score.
+
+![Initial Google PageSpeed Insights score for savaslabs.com. Several significant problems exist!]({{ site.url }}/assets/img/blog/pagespeed-insights-initial.jpg)
+<span class="caption">Yikes!</span>
+
+Going off of these recommendations and adding a few things of our own, we ended
+up with a nice to-do list:
+
+1. Minify our main CSS file, use autoprefixer to add vendor prefixes (since this
+will be required for the soon-to-come Bourbon v5.0) and inline CSS that's
+critical to above-the-fold content
```

Comment:
The way I read this is a little confusing to me (probably given my lack of experience in this realm). We want to minify CSS and add vendor prefixes, but what are we doing with inline CSS? Converting to using inline CSS?